functions/test/firebase/firestore_test.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "helper"
+
+describe "functions_firebase_firestore" do
+  include FunctionsFramework::Testing
+
+  let(:source) { "my-resource" }
+  let(:type) { "type" }
```

Comment:
For realism, go ahead and use `//firestore.googleapis.com/projects/project-id/databases/(default)/documents/gcf-test/12345678` for the source, and `google.cloud.firestore.document.v1.written` for the type.

Reply:
Thanks!